player-height.ts

```diff
@@ -1,18 +1,22 @@
-import {Component, Type} from '@wonderlandengine/api';
+import {Component} from '@wonderlandengine/api';
+import {property} from '@wonderlandengine/api/decorators.js';
 
 /**
  * Set player height for a Y-offset above the ground for
  * 'local' and 'viewer' reference spaces.
  */
 export class PlayerHeight extends Component {
     static TypeName = 'player-height';
-    static Properties = {
-        height: {type: Type.Float, default: 1.75},
-    };
+
+    @property.float()
```

Comment:
You need to specify the default as a parameter to `float()` also, otherwise the UI cannot know about it.

(@DavidPeicho unless there is something here that I don't know?)

Reply:
Should defaults to 0.0 in the editor

Reply:
Better to be safe then sorry, so I added the 0.0 to the float().